playground/nuxt.config.ts

```diff
@@ -2,4 +2,7 @@ import NuxtGraphqlCodegen from "..";
 
 export default defineNuxtConfig({
   modules: [NuxtGraphqlCodegen],
+  graphqlCodegen: {
+    configPath: "./config/codegen.yml",
```

Comment:
Omitting the `configPath` still works in my testing, but it would be nice to have proper testing for the different options.